src/databricks/labs/ucx/install.py

```diff
@@ -807,6 +807,24 @@ def latest_job_status(self) -> list[dict]:
                 continue
         return latest_status
 
+    def repair_run(self, workflow):
+        try:
+            job_id_list = [job_id for step, job_id in self._state.jobs.items() if workflow == step]
+            if job_id_list:
+                job_id = job_id_list[0]
+                job_runs = list(self._ws.jobs.list_runs(job_id=job_id, limit=1))
+                state = job_runs[0].state
+                if job_runs and state.result_state.value != "SUCCESS":
+                    run_id = job_runs[0].run_id
+                    self._ws.jobs.repair_run(run_id=run_id, rerun_all_failed_tasks=True)
+                    logger.info("Exception")
```

Comment:
What is this?

Reply:
Added for testing.Removed now from the code